cmd/contrib/main.go

```diff
@@ -180,19 +180,27 @@ func newPlugin() error {
 		}
 	}
 
-	// As a placeholder, assume the field name is the short version (max 7 chars) of the credential name, starting
-	// from the last word. For example:
+	// As a placeholder, assume the field name is the short version (max 7 chars) of the credential name, starting from the last word.
+	//
+	// When the last word of the credential name is greater than seven characters, the last word is used as the field name.
+	//
+	// For example:
 	// "Personal Access Token" => "Token"
 	// "Secret Key" => "Key"
 	// "API Key" => "API Key"
```

Comment:
I am not sure that this case is still covered. 

To make sure that we are covering all cases, could we extract this functionality into a separate function and write a quick table-test for it? That would make it easy to see which cases are covered and which are not.

Reply:
Sorry to surface this PR after a really long time! Just had some time to look at this.

You were right, @jpcoenen. `API Key` case wasn't covered. I have made some changes in https://github.com/1Password/shell-plugins/pull/263/commits/aecdd2dd526e04e5ed33ba0a29d810b0fe8ef78f that should cover this case, and it includes a test to ensure that all cases are successful too. Would like your feedback!